.gitignore

```diff
@@ -4,3 +4,4 @@ google-service-account.json
 __pycache__
 *.DS_Store
 staticfiles
+.vscode/*
```

Comment:
FYI, you can use a host-wide [$XDG_CONFIG_HOME/git/ignore](https://git-scm.com/docs/gitignore) to ignore patterns across all your projects. Any editor-specific ignores are best in there, otherwise we end up with all kinds of patterns from everyones editor for every project, which isn't necessary.

Not a big deal, not many folks know about this git feature.

Reply:
awesome! thanks for the tip!